Probesys/bileto
# new: Add roles

Related to https://github.com/Probesys/bileto/issues/88

Changes proposed in this pull request:

- Remove the `roles` field from the User entity
- Create a Role entity to store custom permissions of the application
- Add index / new / create roles pages and endpoints
- Give a "super" roles to user created with the CLI

How to test the feature manually:

1. Go to http://localhost:8000/roles
2. Check that the "Super-admin" role exists
3. Create some admin and orga roles and check they are appearing
4. Create a user with the CLI, check in the database that an authorization is associated to this user and to the "super" role

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
